src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -225,6 +225,7 @@ protected OptionParser buildOptionParser() {
 		parser.accepts( "exclude_columns", "exclude these columns, formatted as exclude_columns=col1,col2" ).withRequiredArg();
 		parser.accepts( "blacklist_dbs", "ignore data AND schema changes to these databases, formatted as blacklist_dbs=db1,db2. See the docs for details before setting this!").withRequiredArg();
 		parser.accepts( "blacklist_tables", "ignore data AND schema changes to these tables, formatted as blacklist_tables=tb1,tb2. See the docs for details before setting this!").withRequiredArg();
+		parser.accepts( "include_column_values", "include only rows with these values formatted as include_column_values=C=x,D=y").withRequiredArg();
```

Comment:
`c=X,d=y` seems tricky to parse properly (think values that include commas).  Can the options parser accept multiple arguments with the same name?  thinking `--include_column_values=c=X --include_column_values=d=Y` would be superior here if possible/easy.

Reply:
Seems it does support multiple arguments with same name, will try to make the change.

Reply:
It is supported if we use argument, but how should we handle it when we use properties file? Since properties file does not support multiple properties with same name. Any good idea?

Reply:
hmmm, good point.  I guess leave it comma separated.

Reply:
OK, I keep this unchanged.